specification/source/cxf.rst

```diff
@@ -0,0 +1,290 @@
+   hasInput                        Block             InputConnector   Used to define an input connector for a 
+                                                                      block.
+   hasOutput                       Block             OutputConnector  Used to define an output connector for a 
+                                                                      block.
+   hasParameter                    Block             Parameter        Used to define a parameter for a block.
+   hasConstant                     Block             Constant         Used to define a constant for a block.
+   hasInstance                     Block             Block,           Used to define an instance (input, 
+                                                     InputConnector,  output, parameter or constant) of a 
+                                                     OutputConnector, block.
+                                                     Parameter, 
+                                                     Constant
+   hasFmuPath                      ExtensionBlock    String           Used to include (local or over network)
+                                                                      path to a Functional Mockup Unit
+                                                                      implementation of an extension block.
+   isOfDataType                    InputConnector,   DataType         Used to define the data type for input 
+                                   OutputConnector,                   connectors, output connectors, parameters,
+                                   Parameter,                         and constants.
+                                   Constant
+   containsBlock                   Block             Block            Used in composite block to include other
+                                                                      blocks.
+   connectTo                       OutputConnector,  InputConnector,  Used to connect the output of one block
+                                   InputConnector    OutputConnector  to the input of a block. Only connectors
+                                                                      that carry the same data type can be
+                                                                      connected.
+   translationSoftware             Package, Block    String           Used to include the name of the software
+                                                                      used to CXF representation of the
+                                                                      sequence.
+   translationSoftwareVersion      Package, Block    String           Used to include the version of the
+                                                                      software used to CXF representation of
+                                                                      the sequence.		
```

Comment:
Fix english grammar.

Reply:
@anandkp92 You need to correct the grammar. "Used to include the version of the software used to CXF representation of the sequence" is not English. The same comment applies for the entry above.

Also, make "connectors, Parameters and Constants." consistent. Either use the technical terms (preferred), in which case it is InputConnector and OutputConnector, or use lower case for all.

Moreover, it is not "(local or over network)" but "(local or on the network)" as communication is over the network but the file is on the network.